### PR TITLE
Building front end assets with composer.

### PR DESCRIPTION
**Pull request by grasmash**

### CODE REVIEW

**Comment by grasmash** [comment]
Hey hey! It worked!

**Comment by danepowell** [comment]
I think `composer.lock` needs to be updated

**Comment by danepowell** [comment]
Matt points out that even if composer.lock were updated, this upstream issue would be a blocker:
https://github.com/francoispluchino/composer-asset-plugin/issues/7

File: composer.json [hunk 1]
```diff
@@ -137,6 +140,11 @@
         "drupal/entity_browser": "8.1.0-alpha3",
         "drupal/views_infinite_scroll": "8.1.0",
         "drupal/media_entity_document": "^8.1.0",
-        "drupal/video_embed_field": "^8.1.0"
+        "drupal/video_embed_field": "^8.1.0",
+        "fxp/composer-asset-plugin": "dev-master#b2cc94c8018a7f4d61d1e0ab0d211a2713daa4fe",
```
Comment (balsama) [inline composer.json]: Composer doesn't let you pull specific commit hashes for dependencies of dependencies. So when lightning is a dependency of BLT or Lightning Project, this won't work.

Can you pull 1.1.4 and patch whatever functionality is missing?

**Pull request closed without merging**